sdk/src/identity/tests/claim_aggregation/validation.rs

```diff
@@ -739,6 +739,7 @@ async fn unsupported_did_method() {
 // TO DO (CAI-7996): Not sure why this doesn't run on Wasm/WASI.
 #[cfg(not(target_arch = "wasm32"))]
 #[tokio::test]
+#[ignore] // TODO: this is timing out every time it runs.
```

Comment:
Ran into this as well.

Reply:
Ok. It is generally better to file an issue on github so we can track it there, but I'll update this.

Reply:
FYI, I had already filed an issue here: 
https://github.com/contentauth/c2pa-rs/issues/1277
I'd rather track open source issue with github issues